client/src/components/Donation/utils.ts

```diff
@@ -0,0 +1,10 @@
+const numToCommas = (num: number) =>
+  num.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
+
+// the number is used to indicate to the doner about how much hours of free education their dontation will provide.
+const contributedHoursOfFreeEduction = 50;
```

Comment:
```suggestion
const EDUCATION_HOURS_PER_DOLLAR = 50;
```
It's a constant so we might as well make that clear. Also, I think this is more explicit.